recipes/atomicfile/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "atomicfile" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 190502c5c0b1145f5c294e0be07a9f029d8ea728a0d0806e0ca466467208763e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - atomicfile
+
+about:
+  home: https://github.com/sashka/atomicfile-py
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.rst
```

Comment:
If you don't mind you could ask upstream whether they could include the license in the future. Not terribly important but they might not be aware of it.

Reply:
Sure, I can open a pull request.

Reply:
I created the patch 

```
From 7c1f3b35fe0e09835ce86508f7fcdfb18a252a05 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sat, 16 May 2020 17:55:34 +0200
Subject: [PATCH] include license in pip Package

---
 MANIFEST.in | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 MANIFEST.in

diff --git a/MANIFEST.in b/MANIFEST.in
new file mode 100644
index 0000000..43a950e
--- /dev/null
+++ b/MANIFEST.in
@@ -0,0 +1 @@
+include LICENSE.rst
```

Unfortunately the repository is archived/ read-only so I can not push it.

Reply:
I see. Nevermind.